hnn_core/gui/gui.py

```diff
@@ -1015,6 +1019,58 @@ def _get_evoked_widget(name, layout, style, location, data=None,
     return drive, drive_box
 
 
+def _get_tonic_widget(name, layout, style, data=None):
+
+    default_data = {
+        'amplitude': 0,
+        't0': 0,
+        'tstop': 0,
+        'seedcore': 14,
+    }
+
+    cell_types = ['L2_basket', 'L2_pyramidal', 'L5_basket', 'L5_pyramidal']
+
+    if isinstance(data, dict):
+        default_data.update(data)
+    kwargs = dict(layout=layout, style=style)
+
+    amplitudes = dict()
+    for cell_type in cell_types:
+        if cell_type in data:
+            default_data.update(data[cell_type])
+        amplitudes[cell_type] = BoundedFloatText(
+            value=deepcopy(default_data['amplitude']),
+            description=cell_type, min=0, max=1e6, step=0.01, **kwargs)
+
+    start_times = BoundedFloatText(
+        value=deepcopy(default_data['t0']), description="Start time",
+        min=0, max=1e6, step=0.01, **kwargs)
```

Comment:
```suggestion
        min=0, max=1e6, step=1.0, **kwargs)
```
This is a pretty small step size considering what's being changed

Reply:
Done